remotespark/livyclientlib/livysession.py

```diff
@@ -50,8 +51,10 @@ def __init__(self, http_client, properties, ipython_display,
         self._status_sleep_seconds = status_sleep_seconds
         self._statement_sleep_seconds = statement_sleep_seconds
         self._create_sql_context_timeout_seconds = create_sql_context_timeout_seconds
+        session_guid = uuid.uuid4()
 
-        self._state = LivySessionState(session_id, self._http_client.connection_string,
+
+        self._state = LivySessionState(session_guid, session_id, self._http_client.connection_string,
```

Comment:
Yes, it should be a unique for every session created. It'll help us to identify all of the related events to specific session instead of using session Id which can be similar with other ones.